sounds/models.py

```diff
@@ -390,7 +390,7 @@ class Sound(SocialModel):
 
     # counts
     num_comments = models.PositiveIntegerField(default=0)  # Updated via django (sound view and delete comment view)
-    num_downloads = models.PositiveIntegerField(default=0)  # Updated via database trigger
+    num_downloads = models.PositiveIntegerField(default=0, db_index=True)  # Updated via database trigger
     avg_rating = models.FloatField(default=0)  # Updated via database trigger
     num_ratings = models.PositiveIntegerField(default=0)  # Updated via database trigger
```

Comment:
All these fields have comments "updated via database trigger" which is no longer true right? Or is it?

Reply:
good point, fixed